content/cfp.html

```diff
@@ -82,6 +82,6 @@
 <p>We’ll publish the list of selected talks as soon as we can after the deadline.</p>
 
 <h2 id="financial-support">Financial support for speakers</h2>
-<p>Grants to assist speakers on low budgets will be available, as well as free and discounted tickets. They'll be allocated by the committee. Any speaker who needs help will receive it. <a href="/financial-aid/">Click here</a> to apply for financial aid. Financial aid applications close when the CFP closes.</p>
+<p>Grants to assist speakers on low budgets will be available, as well as free and discounted tickets. They'll be allocated by the committee. Any speaker who needs help will receive it. <a href="/financial-aid/">Apply for financial assistance.</a> Financial aid applications close when the CFP closes.</p>
```

Comment:
We use "financial aid" elsewhere on the site.  I'm not bothered either way, but if you want to be consistent, you could make changes elsewhere.  @Lukasa may have opinions on this.